README.rst

```diff
@@ -20,93 +20,95 @@ How to get started
 ------------------
 
 We assume you have a good understanding of (unix)ODBC. If not, make sure you
-read their documentation carefully - there are lot's of traps to step into.
-
-Get the EXASolution database
-````````````````````````````
-
-If you do not have access to an EXASolution database, download EXASolo for free
-from EXASOL: http://www.exasol.com/en/test-drive/
-
-The database is a VM image. You will need VirtualBox, VMWare Player, or KVM to
-run the database. Start the database and make sure you can connect to it as
-described in the How-To from EXASOL.
+read their documentation carefully - there are lot's of traps 🪤 to step into.
 
 Meet the system requirements
 ````````````````````````````
 
 On Linux/Unix like systems you need:
 
-- the packages unixODBC and unixODBC-dev >= 2.2.14
-- Python >= 2.7
-- Download and install the ODBC client drivers from EXASOL >= 5
-- configure ODBC.ini and ODBCINST.ini
+- An `Exasol DB <test_drive_>`_
+
+  - >= 7.1.6
+  - >= 7.0.16
+
+- The packages unixODBC and unixODBC-dev >= 2.2.14
+- Python >= 3.6
+- The Exasol `ODBC driver <odbc_driver_>`_
+- The ODBC.ini and ODBCINST.ini configurations files setup
 
 Turbodbc support
 ````````````````
 
-- Turbodbc and sqlalchemy_exasol as well do now support python 2.7, 3.4 and 3.6.
+- You can use Turbodbc with sqlalchemy_exasol if you use a python version >= 3.8.
```

Comment:
is python>=3.8 because TurboODBC requires this, or what is the reason

Reply:
Good point, rechecked `turbodbc 3.3.0` still seems to support older versions (mistakenly checked against the current version). Still I think we would say `>=3.6` because 3.6 is what we currently test against right?